src/api/blocks.ts

```diff
@@ -1,5 +1,7 @@
 import qs from 'qs';
-import { PagedResult, PagedParams, fetchInfo } from '.';
+import { PagedResult, PagedParams, Info } from '.';
+import { AppEnv } from '../app-env';
```

Comment:
For some reason import ...  '~app-env' fails to build.

Reply:
Hmm, probably would need to checkout first and investigate. However, if you agree with injecting via webpack, it gets obsolete...

Reply:
Maybe you give it a go? Otherwise, I'd of course be fine to do it as well, as long as you agree of course...